### PR TITLE
user guide for docker

### PR DESCRIPTION
**Pull request by madhur-tandon**
Hmm, deployment to docker is pretty straightforward and small. Thus, this page / UG is also small.

I am not sure what more stuff can be added really, any ideas perhaps?

It doesn't make sense to list the configurable parameters here since we already have object references for it + it is easily accessible through `-h`

Let me know what you think about this for UG Docker.

### CODE REVIEW

File: content/docs/user-guide/deploying/docker.md [hunk 1]
```diff
@@ -10,14 +14,50 @@ pip install mlem[docker]
 pip install docker
 ```
 
-## TODO
+## Description
+
+Deploying to a docker container involves 2 main steps:
+
+1. Build the docker image.
```
Comment (aguschin) [inline content/docs/user-guide/deploying/docker.md:21]: Link to user-guide/building/docker

File: content/docs/user-guide/deploying/docker.md [hunk 2]
```diff
@@ -10,14 +14,50 @@ pip install mlem[docker]
...
+2. Start the server inside the image.
```
Comment (aguschin) [inline content/docs/user-guide/deploying/docker.md:22]: Mention that you runs the image, to be more precise?

Comment (aguschin) [reply content/docs/user-guide/deploying/docker.md:22]: Maybe "(by running `docker run` under the hood)"

File: content/docs/user-guide/deploying/docker.md (hunk 1 above)
Comment (aguschin) [reply content/docs/user-guide/deploying/docker.md:21]: + maybe "(by running `docker build` under the hood)"

File: content/docs/user-guide/deploying/docker.md [hunk 3]
```diff
@@ -10,14 +14,50 @@ pip install mlem[docker]
...
+```
+CONTAINER ID   IMAGE                           COMMAND                  CREATED          STATUS          PORTS     NAMES
+0aa976159580   mlem-deploy-1666728279:latest   "/bin/sh -c 'sh run.…"   26 seconds ago   Up 25 seconds             mlem-deploy-1666728455
+...
+```
+
+### Configurable parameters
+
+A lot of parameters can be configured, the full list of which can be accessed
+using `mlem deploy run docker_container -h`
```
Comment (aguschin) [inline content/docs/user-guide/deploying/docker.md:63]:
```suggestion
using `mlem deploy run docker_container -h`.
```

**Review by aguschin — APPROVED** [review]
LGTM! This page indeed should be short.

File: content/docs/user-guide/deploying/docker.md [hunk 4]
```diff
@@ -10,14 +14,50 @@ pip install mlem[docker]
...
+⏳️ Loading model from model.mlem
+🛠 Creating docker image mlem-deploy-1666728279
+  🛠 Building MLEM wheel file...
+  💼 Adding model files...
+  🛠 Generating dockerfile...
+  💼 Adding sources...
+  💼 Generating requirements file...
+  🛠 Building docker image mlem-deploy-1666728279:latest...
+  ✅  Built docker image mlem-deploy-1666728279:latest
+✅  Container mlem-deploy-1666728455 is up
```
Comment (aguschin) [inline content/docs/user-guide/deploying/docker.md:38]: FYI, created this issue https://github.com/iterative/mlem/issues/449

**Comment by aguschin** [comment]
> ^^guess the deployment failed, since it is inaccessible, hence the errors about links

did you rebased (or merged #188 in this PR)? I see some links you fixed already in #188

File: content/docs/user-guide/deploying/docker.md (hunk 1 above)
Comment (madhur-tandon) [reply content/docs/user-guide/deploying/docker.md:21]: done

File: content/docs/user-guide/deploying/docker.md (hunk 2 above)
Comment (madhur-tandon) [reply content/docs/user-guide/deploying/docker.md:22]: done

File: content/docs/user-guide/deploying/docker.md (hunk 3 above)
Comment (madhur-tandon) [reply content/docs/user-guide/deploying/docker.md:63]: done

**Pull request merged**